pages/vi/vi-nation.md

```diff
@@ -12,15 +12,16 @@ In [Step 1 - BeLL Installation - Configuration](vi-configurations.md) you regist
 
 There should be constant communication between the nation and the communities. While it is not necessary for remote communities in the field, it is ideal for our goals of "improving the software and testing the increasing forms of communication and feedback between the nation and the communities". This communication takes the form of a syncing process from the community side, where you select material to send to the nation.
 
-Click [here](http://127.0.0.1:5985/apps/_design/bell/MyApp/index.html) to access your Community BeLL.
+Make sure vagrant is running and then click [here](http://127.0.0.1:5985/apps/_design/bell/MyApp/index.html) to access your Community BeLL.
 
 **NOTE**: After you register your community, but before you can sync with the nation, you need to create an additional dummy user on your community. Therefore, create a quick additional user under "Become a Member" on the login page (HINT: When creating the dummy user, don't give it a password that you actually use, as other people logged in as admin may be able to see it-- you won't need the password in the future so don't worry about having to remember or save it after this one time). Then, login and double-check that you're listed under Members. Then, log out and log back in with your admin account. Now that your community has a user, you can sync with the nation.
 
 ![Clicking on "Dummy User"](images/vi-nation-add-dummy.png "Dummy User")
 
 ## Sync With the Nation
 
-In [Step 4](vi-bell-apps.md#Different_Kinds_of_Updates_to_Your_Community), you did 3 kinds of updates that you receive on the community side: updates, publications, and surveys. Another important type of update is the reverse: syncing with the nation. This sends data about your community to the nation.
+
```

Comment:
why adding new line?

Reply:
@mappuji  I thought It would be nice to remind the intern to make sure the vagrant is running, otherwise, the link will break. That is why I added a new line.

Reply:
@mappuji  I thought It would be nice to remind the intern to make sure the vagrant is running, otherwise, the link will break. That is why I added a new line.

Reply:
oh that's cool

Reply:
@amanuely I think mappuji means why did you add the extra blank line at the end of the file. The extra comment about vagrant is helpful though